### PR TITLE
US127772 - Discussions - A11y Tabbing order for Sort and Filter button should be switched

### PR DESCRIPTION
**Pull request by cloudislife8899**
image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1115,6 +1115,7 @@
         "@brightspace-ui-labs/caketray": "^1",
         "@brightspace-ui-labs/checkbox-drawer": "github:BrightspaceUILabs/checkbox-drawer#semver:^1",
         "@brightspace-ui-labs/list-item-accumulator": "^1",
+        "@brightspace-ui-labs/pagination": "^1.0.2",
```
Comment (cloudislife8899) [inline package-lock.json:1118]: I did not make changes to this file. Not sure why it is displaying made changes. Should I revert to the old version?

Comment (JamieTran) [reply package-lock.json:1118]: This shows up when you do an `npm install` and it finds dependencies that should be updated or added. This looks okay to merge in.

File: components/left-panel/discussions/consistent-evaluation-evidence-discussion.js [hunk 2]
```diff
@@ -41,22 +41,28 @@ export class ConsistentEvaluationEvidenceDiscussion extends SkeletonMixin(RtlMix
 
 	static get styles() {
 		return css`
+			.d2l-consistent-evaluation-evidence-discussion-list-modifiers {
+				justify-content: flex-end;
+				display: flex;
+			}
+			:host([dir="rtl"]) .d2l-consistent-evaluation-evidence-discussion-list-modifiers {
+				display: flex;
```
Comment (JamieTran) [inline components/left-panel/discussions/consistent-evaluation-evidence-discussion.js:49]: do we need `display: flex;` here? also what about `justify-content: flex-start;`

**Review by JamieTran — APPROVED** [review]

**Comment by JamieTran** [comment]
LGTM!

**Comment by amauryperez** [comment]
good job michael

**Review by amauryperez — APPROVED** [review]

**Pull request merged**